### ISSUE TITLE
Add src/web-ext-artifacts/* to .gitignore?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blurts-addon#72]
I noticed we have https://github.com/mozilla/blurts-addon/tree/master/src/web-ext-artifacts in the repo, which currently includes a generated ZIP file.
Not sure if src/web-ext-artifacts should be added to the .gitignore since it's a generated asset.

**Comment by pdehaan** [comment]
Actually, maybe that's an old artifact and should just be killed. If I do a fresh <kbd>$ npm install</kbd> and <kbd>$ npm run build</kbd>, it looks like the web-ext artifact gets saved to the "./dist/firefox_monitor-0.01.zip" folder (and not "./src/web-ext-artifacts/", which is what is committed to the repo).

```sh
$ npm run build

> blurts-addon@1.0.0 prebuild /Users/pdehaan/dev/github/mozilla/blurts-addon
> npm run bundle-utils && npm run setup-breaches


> blurts-addon@1.0.0 bundle-utils /Users/pdehaan/dev/github/mozilla/blurts-addon
> copyStudyUtils src/privileged


> blurts-addon@1.0.0 setup-breaches /Users/pdehaan/dev/github/mozilla/blurts-addon
> node scripts/setup-breaches.js && node_modules/svgo/bin/svgo -q src/PwnedLogos/*.svg


> blurts-addon@1.0.0 build /Users/pdehaan/dev/github/mozilla/blurts-addon
> web-ext build

Applying config files: ./package.json, ./web-ext-config.js
Building web extension from /Users/pdehaan/dev/github/mozilla/blurts-addon/src
Your web extension is ready: dist/firefox_monitor-0.01.zip
```

**Comment by nhnt11** [comment]
Fixed by 76620cc
